### ISSUE TITLE
CSS not showing in Nuxt Production

### ISSUE THREAD

**Issue by santicros** [shentao/vue-multiselect#1145]
When deploying the project with Nuxt (npm run build) the styles don't show properly. 
I've tried to use the Nuxt module and didn't work at build, but at dev work perfectly.

**Comment by ardasatata** [comment]
I figured it out after 5 hours trying lol 

Just add this to your css on nuxt.config.js
```
  /*
   ** Global CSS
   */
  css: [
    '~/assets/css/tailwind.css',
    '~/assets/css/main.css',
    '~/node_modules/vue-multiselect/dist/vue-multiselect.min.css'
  ],
```

hope this helps!

**Comment by santicros** [comment]
Hey @ardasatata thanks for your comment!
I had tried putting the Global CSS links and didn't work.

I've finally discovered it was because I'm also using TailwindCSS with the nuxt-tailwindcss plugin and the classes were deleted on the Post CSS.

What has finally worked has been adding to my nuxt.config.js in the PostCSS whitelistPatterns section a multiselect value (and also added the nuxt transitions, because didn't work on production):

``` javascript
export default {
  purgeCSS: {
    whitelistPatterns: [
      /-(leave|enter|appear)(|-(to|from|active))$/,
      /^(?!(|.*?:)cursor-move).+-move$/,
      /^nuxt-link(|-exact)-active$/,
      /multiselect/
    ]
  }
}
```
